src/containerapp/azext_containerapp/commands.py

```diff
@@ -32,6 +32,7 @@ def load_command_table(self, _):
     with self.command_group('containerapp replica') as g:
         g.custom_show_command('show', 'get_replica')  # TODO implement the table transformer
         g.custom_command('list', 'list_replicas')
+        g.custom_command('count', 'count_replicas')
```

Comment:
For command which only existing in extension containerapp,  please add tag `is_preview=True` for it.

Reply:
This isn't meant to be a preview command, what do I need to do to make it a part of CLI core? I want it to have the same standing as the other replica commands.

Reply:
I looked through: https://github.com/Azure/azure-cli/pull/27078/files#diff-eebf206e9e6e274c34ea6fea640760d3892a964d339331ce1869a3991024a394 but was hoping there was additional documentation about adding a non-preview command.

Reply:
Hi @Juancpani 
After we have `containerapp` in core, we will only put GA commands in core.
About the iteration process:
Firstly, you need to add new commands or new features in `containerapp` extension, with tag `is_preview=True`. 
After released in extension and being mature enough, you can move it to core, at that time you can remove the `is_preview=True` in extension.

Reply:
Added! Thank you.